johnnycanencrypt/__init__.py

```diff
@@ -341,6 +341,63 @@ def __contains__(self, other: Union[str, Key]) -> bool:
             return False
         return False
 
+    def update_expiary_in_subkeys(
```

Comment:
The spelling here needs to be `expiry` instead of `expiary`. The same error is there in few other places in this PR that need to be changed.